code-studio/src/js/code-studio.js

```diff
@@ -9,6 +9,9 @@
 'use strict';
 /* global Sounds */
 
+// Shim window.console to be safe in IE
+require('./consoleShim')(window);
```

Comment:
This is slightly awkward, but certainly better than modifying `window` as a side effect of requiring the shim ... is there another pattern we could use for this?